docs/_data/navigation.yml

```diff
@@ -0,0 +1,7 @@
+main:
+  - title: "Quick-Start Guide"
+    url: https://github.com/scale-it/algo-builder#installation
+  - title: "API Docs"
+    url: https://scale-it.github.io/algo-builder/ # TODO: Update to custom api readme link
```

Comment:
please create a task for this in pivotal tracker

Reply:
OK.